src/server/protocol.ts

```diff
@@ -868,6 +866,18 @@ namespace ts.server.protocol {
         format?: "original" | "2020"
     }
 
+    /** The response for a EncodedSemanticClassificationsRequest */
+    export interface EncodedSemanticClassificationsResponse extends Response {
+        body?: EncodedSemanticClassificationsResponseBody
+    }
+
+    /**
+     * Implementation response message. Gives series of text spans depending on the format ar.
+     */
+    export interface EncodedSemanticClassificationsResponseBody {
+        endOfLineState: EndOfLineState;
+        spans: ClassifiedSpan[] | number[];

```

Comment:
This isnt correct.  The response currently from `getEncodedSemanticClassifications` returns type as `ts.Classifications` where spans is `number[]`. 
More over you are referencing `ClassifiedSpan` here which is only in services and not in protocol. 
This doesnt correctly reflect what we return.